06-building-a-program-start-to-finish/solving-the-problem/output/helper.js

```diff
@@ -0,0 +1,120 @@
+/**
+* helper object for InvertedIndex class
+* @object
+*/
+const helpers = {
+  /** validates file input types
+   *
+   * @param  {object} file - file to validate
+   * @return {boolean} - returns true if valid or false if invalid
+   */
+  fileIsValid(file) {
+    if (!file.name.toLowerCase().match(/\.json$/)) {
+      return false;
+    }
+    return true;
+  },
+
+  /** validates content of uploaded file
+   *
+   * @param  {array} file - file to validate content
+   * @return {boolean} returns true if valid or false if invalid
+   */
+  validFileContent: (file) => {
```

Comment:
I believe what you are passing is the content of the file and not the actual file here. Please change your parameter name to reflect such.